nameko/cli/main.py

```diff
@@ -19,9 +22,27 @@ def setup_parser():
     return parser
 
 
+def _replace_env_var(match):
+    env_var, _, default = match.groups()
+    return os.environ.get(env_var, default)
+
+
+def _env_var_constructor(loader, node):
+    value = loader.construct_scalar(node)
+    return re.compile(
+        r'\$\{([a-zA-Z][^}:\s]+)(:([^}]+))?\}'
```

Comment:
I think I would prefer these defined as variables, although in this module rather the constants.py. That allows you to give them readable names, e.g.

``` python
env_var_matcher = re.compile(r'\$\{([a-zA-Z][^}:\s]+)(:([^}]+))?\}')
quoted_env_var_matcher = re.compile(r'.*\$\{.*\}.*')) 
```